dts/bindings/sensor/sensor-device.yaml

```diff
@@ -17,3 +17,18 @@ properties:
       This property is defined in the Generic Sensor Property Usages of the HID
       Usage Tables specification
       (https://usb.org/sites/default/files/hut1_3_0.pdf, section 22.5).
+
+  reporters:
+    type: phandle-array
+    description: |
+      Sensor input reporter specifiers, used by virtual sensors to synthesize
+      sensor channels from one or more other sensors. For example, a lid angle
+      sensor synthesized from a lid accelerometer and a base accelerometer.
+
+  reporter-names:
+    type: string-array
+    description: Provided names of sensor input reporter specifiers
+
+reporter-cells:
+  - interval
+  - sensitivity
```

Comment:
I wanted to start another thread for this. I'm not sold that the DT should have this information on a per reporter basis. As a developer I would want to just specify that I want a lid angle sensor with an interval of 60Hz and sensitivity of 5 degrees and use 2 reporting accelerometers. The driver implementation should know how to configure the reporters in order to provide me with what I need.